### DISCUSSION TITLE
Feature request: More lists (i always run out)

### DISCUSSION THREAD

**Discussion by devinschumacher** [community/community#127283] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Would love to have a larger limit on the number of lists you can create.

I also run out. 

They are also very inconvenient to search through later when trying to find things.

ty!

**Comment by Ahmadtaha1122** [comment]
Ok

**Comment by Laneraverfa** [comment]
([L](R)t[R](L))

**Reply by Laneraverfa** [reply]
Harh7@

**Comment by eliancheng** [comment]
I understand the need for a larger limit on lists and improved search functionality; this would definitely enhance usability

**Reply by SalissouSalhaHassan** [reply]
- > ![20240817_110918.jpg](https://github.com/user-attachments/assets/5f2e0e14-5f63-4391-a917-26508509d67d)

**Reply by SalissouSalhaHassan** [reply]
Bonjour

**Reply by siruschavoshi** [reply]
Hi

**Reply by superboyam** [reply]
> - > ![20240817_110918.jpg](https://github.com/user-attachments/assets/5f2e0e14-5f63-4391-a917-26508509d67d)
> 
>

**Reply by MarketBeatAd** [reply]
Good Day

**Comment by Arev55** [comment]
#- [x] - - - - - - - [x] [_@**Arev55**_]()_**_[~~~~````
```

```
````~~~~]()_**_

**Comment by OGherc86** [comment]
I'm over it but more would be great

**Reply by MarketBeatAd** [reply]
You want more

**Comment by siruschavoshi** [comment]
> Ok

**Comment by Gipsy9** [comment]
What its going i dont now befor i die i want help my contare peoppl my famly thinking i get crazy

**Comment by MarketBeatAd** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> Would love to have a larger limit on the number of lists you can create.
> 
> I also run out. 
> 
> They are also very inconvenient to search through later when trying to find things.
> 
> ty!

**Comment by wwgamorcom** [comment]
qqwweerrttyyuuiioo1p1999@gmail.com

**Reply by wwgamorcom** [reply]
- [ ] **![Uploading was (5).pdf …]()
> > qqwweerrttyyuuiioo1p1999@gmail- #```

``` - [x] .com**~~~~****~~~~

**Reply by Shadowsifter** [reply]
> - [ ] **![Uploading was (5).pdf …]()
> > > qqwweerrttyyuuiioo1p1999@gmail- #```
> 
> ``` - [x] .com**~~~~****~~~~

**Comment by Adamu1556** [comment]
real Adam

**Comment by Haidarnassaraldeen** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> Would love to have a larger limit on the number of lists you can create.
> 
> I also run out. 
> 
> They are also very inconvenient to search through later when trying to find things.
> 
> ty!

**Comment by VahidRafieiPour** [comment]
صرافی نوبیتکس براش میلیاردی بیتدکوین واریز کردم آنقدر زیاد خست حسابش در دستم نیست ولی بالغ بر چندین میلیارد تماس میگیرم میگن هش تراکنش خود رو برامون بفرست منم تز 8ود شرکت بیت کویین جواب فرستادم باز میگن این این نیست اول که داشتن منکر میشدن تو اینجا واریزی داشتی صدای ظبن شده تلفنی اپراتورشوم هست که تو استخر عرضی داری اما یک ریال از این بیت کویین های دریافتی رو به حساب من واریز نکردن من چند تا واریزی یک ملیون بیت کویین دارم براشون ریختم اگر این آدرس عرض مشکل داشت چرا گرفتین اگر هش تراکنش نداشت نباید قبول میکردین حالا من هش تراکنش در قالب یه نرم افزار که شرکت بیت کوین برام ارسال کرده براشون فرستادم دانش این رو حدا ن ازش استخراج کنن نزدیک بالای 4الی 7 میلیاردی شایدم بیشتر کیف پول من بود سه بار سایت خودشون تو این چند رپر اخیر تاثیر دادن و تو آخرین بروز رسانی کیف پوزثر کن گم شده و حال. نیستن پول منو بهم لثبدن با اینکه سند قانونی از شرکت بیت کویین دارم که پلیس در 14 صفحه اونو تایید کرد الان باید چیکارکنم

**Comment by VahidRafieiPour** [comment]
![BTC_transaction_report.pdf](https://github.com/user-attachments/files/17796201/BTC_transaction_report.pdf)

آیتم سند ارسالی شرکت بیت کویین که مختص من چون من از ماکروسافت درقبال تبلیغات که براشون انجام میدم بهم یه تثنزم افزار داده مه ازش ادرس،بیت کویین استخراج کنم اصلا هش مگه ایزدی ادرس من نیست آیتم ادرس منه که براشون عرض واریز کردم میگن این قبول نیست دارن کلاه برداری مبکننن

**Comment by Bedminator** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> Would love to have a larger limit on the number of lists you can create.
> 
> I also run out. 
> 
> They are also very inconvenient to search through later when trying to find things.
> 
> ty!
